BaseFilter/sections/antiadblock.txt

```diff
@@ -3070,7 +3070,7 @@ mixfaucet.com#%#//scriptlet("abort-on-property-read", "adblockDetector")
 @@||cloudfront.net/?*=$domain=1cloudfile.com
 !#endif
 ! For DNS filtering
-||1cloudfile.com$replace=/Request\("https:\/\/d192r5l88wrng7\.cloudfront\.net"/Request\("https:\/\/example\.com"/
+||1cloudfile.com$replace=/("https:\/\/d192r5l88wrng7\.cloudfront\.net"/("https:\/\/example\.com"/
```

Comment:
Suggested another solution in PM, because this does not help.

Reply:
@jellizaveta 

![image](https://user-images.githubusercontent.com/8361299/210637357-665ebff9-24ea-468c-957f-2633674f1678.png)

Reply:
@jellizaveta